public/js/components/AtomEdit/AtomEdit.js

```diff
@@ -1,9 +1,54 @@
-import React from 'react';
+import React, {PropTypes} from 'react';
+
+class AtomEdit extends React.Component {
+
+  static propTypes = {
+    routeParams: PropTypes.shape({
+      atomType: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired
+    }).isRequired,
+    atomActions: PropTypes.shape({
+      getAtom: PropTypes.func.isRequired,
+      updateAtom: PropTypes.func.isRequired
+    }).isRequired,
+    atom: PropTypes.object
```

Comment:
Can we define a PropType for this? only needs to be the base items e.g. id and type